src/main/java/quickcache/model/flashcard/FlashcardPredicate.java

```diff
@@ -39,6 +48,10 @@ public static FlashcardPredicate prepareOnlyTagsFlashcardPredicate(Set<Tag> tags
 
     @Override
     public boolean test(Flashcard flashcard) {
+        if (flashcard == null) {
+            return false;
+        }
```

Comment:
should this return an exception instead? because if the flashcard is `null`, means that somehow the flashcard list contains a `null` and throwing an exception might be more helpful to find out the root of the problem

Reply:
Yeah I think that makes sense, thank you for the suggestion!